lib/example_app/object_types/project_manager.ex

```diff
@@ -2,11 +2,14 @@ defmodule ExampleApp.ObjectTypes.ProjectManager do
   use Ecto.Schema
   import Ecto.Changeset
   alias ExampleApp.Users.User
-  # use ExampleApp.Schema
-
 
+  @primary_key false
   schema "project_managers" do
-    belongs_to :user, User, foreign_key: :wf_user_id, type: :string, primary_key: true
+    belongs_to :user, User,
+      foreign_key: :wf_user_id,
+      type: :string,
+      primary_key: true,
+      references: :wf_user_id
```

Comment:
in `:references` we show ecto which key to use in the user schema when building associations